dozer/cogs/moderation.py

```diff
@@ -56,6 +54,7 @@ class Moderation(Cog):
                     await orig_channel.send("Please configure modlog channel to enable modlog functionality")
 
     async def perm_override(self, member, **overwrites):
+        """Applies permission overrides"""
```

Comment:
Specifically, it applies the given overrides to the given member in their guild.